larq_compute_engine/mlir/transforms/optimize_patterns_common.td

```diff
@@ -11,6 +11,30 @@ def HasOneUse : Constraint<CPred<"$0.hasOneUse()">>;
 
 class ConstantValue<string val> : AttrConstraint<CPred<"IsConstantValue($_self, " # val # ")">>;
 
+def : Pat<(LQ_QuantizeOp
+              (TFL_GreaterEqualOp:$ge_op
+                  $input,
+                  (ConstantOp:$threshold ConstantValue<"0.0f">))),
+          (LQ_QuantizeOp $input),
+          [(HasOneUse $ge_op), (HasOneUse $threshold)],
```

Comment:
I think since this is a constant value we don't care about the usage of it, or do we?
```suggestion
                  (ConstantOp ConstantValue<"0.0f">))),
          (LQ_QuantizeOp $input),
          [(HasOneUse $ge_op)],
```